install.sh

```diff
@@ -120,27 +121,27 @@ ESCAPED_HOST_FILE=${HOST_FILE////\\/}
 if [ "$BROWSER_NAME" == "Chrome" ] || \
    [ "$BROWSER_NAME" == "Chromium" ] || \
    [ "$BROWSER_NAME" == "Vivaldi" ]; then
-  if [ ! -f "$DIR/chrome-host.json" ] || [ ! -f "$DIR/chrome-policy.json" ]; then
-    echo "ERROR: '$DIR/chrome-host.json' or '$DIR/chrome-policy.json' is missing."
+  if [ ! -f "$JSON_DIR/chrome-host.json" ] || [ ! -f "$JSON_DIR/chrome-policy.json" ]; then
+    echo "ERROR: '$JSON_DIR/chrome-host.json' or '$JSON_DIR/chrome-policy.json' is missing."
     echo "If you are running './install.sh' from a release archive, please file a bug."
     echo "If you are running './install.sh' from the source code, make sure to follow CONTRIBUTING.md on how to build first."
     exit 1
   fi
-  cp "$DIR/chrome-host.json" "$TARGET_DIR/$APP_NAME.json"
+  cp "$JSON_DIR/chrome-host.json" "$TARGET_DIR/$APP_NAME.json"
   mkdir -p "$TARGET_DIR"/../policies/managed/
-  cp "$DIR/chrome-policy.json" "$TARGET_DIR/../policies/managed/$APP_NAME.json"
+  cp "$JSON_DIR/chrome-policy.json" "$TARGET_DIR/../policies/managed/$APP_NAME.json"
 else
-  if [ ! -f "$DIR/firefox-host.json" ]; then
-    echo "ERROR: '$DIR/firefox-host.json' is missing."
+  if [ ! -f "$JSON_DIR/firefox-host.json" ]; then
+    echo "ERROR: '$JSON_DIR/firefox-host.json' is missing."
     echo "If you are running './install.sh' from a release archive, please file a bug."
     echo "If you are running './install.sh' from the source code, make sure to follow CONTRIBUTING.md on how to build first."
     exit 1
   fi
-  cp "$DIR/firefox-host.json" "$TARGET_DIR/$APP_NAME.json"
+  cp "$JSON_DIR/firefox-host.json" "$TARGET_DIR/$APP_NAME.json"
 fi
 
 # Replace path to host
-sed -i -e "s/%%replace%%/$ESCAPED_HOST_FILE/" "$TARGET_DIR/$APP_NAME.json"
+sed -i '' -e "s/%%replace%%/$ESCAPED_HOST_FILE/" "$TARGET_DIR/$APP_NAME.json"
```

Comment:
Unfortunately this doesn't work on linux 😞 Is there a way to make it compatible?

```
sed: can't read : No such file or directory
```

Reply:
Whoops, didn't realize that was a sed BSD-ism. Perhaps instead of first copying the files and then using `-i`, we could just run sed on the original location and output to `NativeMessagingHosts`?

Reply:
sounds like a good approach